src/core/cache_utils.jl

```diff
@@ -2,10 +2,7 @@ const ParamCacheKey = NamedTuple{(:stage, :type, :name), NTuple{3, Symbol}}
 
 make_cache_key(stage, type, name) = (stage = stage, type = type, name = name)
 
-"""
-Priority for keeping data in cache to serve reads.
-Currently unused.
-"""
+# Priority for keeping data in cache to serve reads. Currently unused.
```

Comment:
I don't know why a real docstring fails. Not willing to invest more time at the moment.